src/libopenrave-core/jsonparser/gpgutils.cpp

```diff
@@ -0,0 +1,215 @@
+// -*- coding: utf-8 -*-
+// Copyright (C) 2023 Tan Li Boon <liboon.tan@mujin.co.jp>
```

Comment:
```suggestion
// Copyright (C) 2023 Tan Li Boon <undisputed.seraphim@gmail.com>
```

ridiculously minor but which should be better?